dark/index.html

```diff
@@ -212,18 +212,18 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
      <section id="contact"> 
       <h1>CONTACT ME</h1>
       <h3>CONTACT <span class="text-info">ME</span></h3> 
-      <div class="formContainer my-3 bg-dark p-5 rounded-3">
+      <div class="formContainer my-3 p-5 rounded-3">
         <form class="m-auto">
           <div class="mb-3">
-            <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name"
+            <input type="text" class="form-control text-white" placeholder="Enter Your Name"

```

Comment:
Don't remove the bg-dark of the inputs @Rishav1707 

```suggestion
            <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name"
```